Assets/Scripts/AudioManager.cs

```diff
@@ -16,6 +17,8 @@ public class AudioManager : MonoBehaviour
         private AudioClip mCoinCollection;
         [SerializeField]
         private AudioClip mCoinDeposited;
+        [SerializeField]
+        private AudioClip mEquipVeto;

```

Comment:
go ahead and add place holders for the other audio clips

e..g private AudioClip mEquipMagicBullet

for now you can just set them all to be the same equip sound